src/components/application_manager/include/application_manager/application.h

```diff
@@ -405,6 +405,20 @@ class Application : public virtual InitialApplicationData,
   virtual void UpdateHash() = 0;
 
   /**
+   * @brief Retrieves flag_sending_hash_change_after_awake_
+   * @return Returns TRUE if hashID was changed during suspended state
+   * otherwise returns FALSE.
+   */
+  virtual bool flag_sending_hash_change_after_awake() const = 0;
```

Comment:
@AKalinich-Luxoft 
Why not to call the method like IsHashChangedAfterAwake()? I think flag name is part of implementation which can be changed.